datumaro/plugins/yolo_format/format.py

```diff
@@ -7,3 +7,21 @@ class YoloPath:
     DEFAULT_SUBSET_NAME = "train"
     SUBSET_NAMES = ["train", "valid"]
     RESERVED_CONFIG_KEYS = ["backup", "classes", "names"]
+    LABELS_EXT = ".txt"
+    SUBSET_LIST_EXT = ".txt"
+
+
+class Yolo8Path(YoloPath):
```

Comment:
Official name is YOLOv8 (Yolo version 8).

It looks it isn't important, but I would to have users here when they a looking for YOLOv8 support. It is critical for search engine optimization.

Let's try to add v8 instead of 8 anywhere.

Reply:
Would it be enough too have YOLOv8 in the docs, or would it be better to also rename all the formats, like yolo8_segmentation -> yolo_v8_segmentation ?

Reply:
I would rename to yolov8_segmentation.
Also, I would recommend to refactor code as well, to be consistent.

Reply:
renamed YOLO8 to YOLOv8 everywhere